### DISCUSSION TITLE
Use Sagemaker with Hydra

### DISCUSSION THREAD

**Discussion by dudulasry** [facebookresearch/hydra#2365] [General]
Hi everyone!
Is there somebody here who uses Sagemaker with hydra? I'd like to discuss a bit about the setup.

Specifically, I can't seem to understand how to pass parameters correctly from my Sagemaker code to my entrypoint script.

**Comment by jieru-hu** [comment]
related post https://stackoverflow.com/questions/73618291/using-sagemaker-with-hydra

cc @dennisbappert
